From 043f60e1a5fd6ca9b1639afabedac8ca32bf6f00 Mon Sep 17 00:00:00 2001
From: Jason Abele <jason@cozybit.com>
Date: Sun, 26 Aug 2012 22:23:40 +0900
Subject: [PATCH] lock feed versions

---
 feeds.conf.default |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/feeds.conf.default b/feeds.conf.default
index 518bf68..ae6af3a 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,6 +1,6 @@
-src-svn packages svn://svn.openwrt.org/openwrt/packages
-src-svn xwrt http://x-wrt.googlecode.com/svn/trunk/package
-src-svn luci http://svn.luci.subsignal.org/luci/trunk/contrib/package
+src-svn packages svn://svn.openwrt.org/openwrt/packages@32583
+#src-svn xwrt http://x-wrt.googlecode.com/svn/trunk/package
+src-svn luci http://svn.luci.subsignal.org/luci/trunk/contrib/package@8846
 #src-svn phone svn://svn.openwrt.org/openwrt/feeds/phone
 #src-svn efl svn://svn.openwrt.org/openwrt/feeds/efl
 #src-svn xorg svn://svn.openwrt.org/openwrt/feeds/xorg
-- 
1.7.9.5

